src/Omega_h_describe.cpp

```diff
@@ -0,0 +1,38 @@
+#include <Omega_h_file.hpp>
+
+
+int main(int argc, char** argv)
+{
+    auto lib = Omega_h::Library(&argc, &argv);
+    Omega_h::Mesh mesh = Omega_h::read_mesh_file(argv[1], lib.world());
+
+    printf("\nMesh Properties:\n");
+    for(int dim=0; dim < mesh.dim(); dim++)
+        printf("(Dim, Num Mesh Entities): (%d, %d)\n", dim, mesh.nents(dim));
```

Comment:
Maybe move the `(Dim, Num Mesh Entities)` format specifier to the heading above?